specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/Clusters.json

```diff
@@ -0,0 +1,668 @@
+            10000,
+            25000,
+            50000
+          ],
+          "x-ms-enum": {
+            "name": "Capacity"
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "The SKU (tier) of a cluster.",
+          "enum": [
+            "CapacityReservation"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterSkuNameEnum",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Identity": {
+      "type": "object",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of managed service identity.",
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "IdentityType",
+            "modelAsString": false
+          }
+        },
+        "userAssignedIdentities": {
+          "description": "The list of user identities associated with the resource. The user identity dictionary key references will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/UserIdentityProperties"
+          }
+        }
+      },
+      "required": [
+        "type"
+      ],
+      "description": "Identity for the resource."
+    },
+    "UserIdentityProperties": {
+      "type": "object",
+      "description": "User assigned identity properties.",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "description": "The principal id of user assigned identity.",
+          "type": "string"
+        },
+        "clientId": {
+          "readOnly": true,
+          "description": "The client id of user assigned identity.",
+          "type": "string"
+        }
+      }
+    },
+    "AssociatedWorkspace": {
+      "type": "object",
+      "description": "The list of Log Analytics workspaces associated with the cluster.",
+      "properties": {
+        "workspaceId": {
+          "readOnly": true,
+          "description": "The id of the assigned workspace.",
+          "type": "string"
+        },
+        "workspaceName": {
+          "readOnly": true,
+          "description": "The name id the assigned workspace.",
+          "type": "string"
+        },
+        "resourceId": {
+          "readOnly": true,
+          "description": "The ResourceId id the assigned workspace.",
+          "type": "string"
+        },
+        "associateDate": {
```

Comment:
>associateDate [](http://example.com/codeflow?start=9&length=13)

naming guidelines for timestamps are to use At suffix, like 'associatedAt' or 'createdAt'.

Reply:
Out of scope of change and a breaking change.